s4e-web/.angulardoc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "repoId": "0798367e-3e97-486a-a4af-60d3997da6a1",
+  "lastSync": 0
+}
```

Comment:
I'm guessing that this is VS file, should it be included in the repository?

Reply:
It shouldn't, good to know that it was pushed...